### PR TITLE
feat: add commands for today's events and teaser

### PR DESCRIPTION
**Pull request by snaeil**
Provides plugin core files and adds the following commands with the respective features:

- `:APeoplesCalendar`: Opens a popup showing today's events
- `:APeoplesCalendarTeaser`: opens a popup for a short time, showing a teaser for a randomly picked event of today

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+A People's Calendar (aPC) is a project that seeks to promote the worldwide history of working class movements and liberation struggles in the form of a searchable "On This Day" calendar.
+
+This repository hosts the aPC neovim plugin, implemented in Lua.
+
+This project is open source and the information contained in our event database is non-proprietary and will _always_ be freely available for users, developers, and the public.
+
+## Install
+
+This example should Using lazy you can use this:
```
Comment (aPeoplesCalendar) [inline README.md]: I think there's a typo here. Did you mean something like `"Here is an example using lazy dependencies:"`?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        require("apeoplescalendar").today_teaser()
+    end,
+}
+```
+
+## Contribute
+
+- A People's Calendar is [open source](https://github.com/aPeoplesCalendar).
+- We are happy about feature requests, issue reports and code submissions
+- Commit messages must correspont to _[Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)_
```
Comment (aPeoplesCalendar) [inline README.md]: Did you mean `correspond` instead of `correspont` here?

**Comment by aPeoplesCalendar** [comment]
Thank you!! Super excited about this.

File: README.md (hunk 2 above)
Comment (snaeil) [reply README.md]: exactly

File: README.md (hunk 1 above)
Comment (snaeil) [reply README.md]: Yes, you are right, thanks for reviewing in detail!

**Comment by snaeil** [comment]
@aPeoplesCalendar updatet `README.md` accordingly

File: README.md (hunk 1 above)
Comment (snaeil) [reply README.md]: Fixed in last commit

File: README.md (hunk 2 above)
Comment (snaeil) [reply README.md]: Fixed in last commit

**Review by aPeoplesCalendar — APPROVED** [review]

**Pull request merged**